Added UI to Set Server URL and removed database.txt file

skaro13 opened the pull request:
I removed the database.txt file and added a server URL query:
<img width="1392" alt="Capture d’écran, le 2019-04-22 à 22 18 54" src="https://user-images.githubusercontent.com/8635747/56548539-5803a880-654e-11e9-97e0-1fdbdf6c0595.png">

cvium commented on main.js:
```diff
@@ -12,8 +14,8 @@
 
-    var enableDevTools = false;
-    var enableDevToolsOnStartup = false;
+    var enableDevTools = true;
```
Was this intentional?

cvium commented on main.js:
```diff
@@ -445,17 +456,17 @@
     function getAppUrl() {
+        var url = getAppBaseUrl();
+
+        if(url){
```
```suggestion
        if (url) {
```

cvium commented on main.js:
```diff
@@ -935,34 +944,14 @@
-            catch(e) {
-                console.log(e);
+ 
+            if(url){
```
```suggestion
            if (url) {
```

cvium commented on firstrun/Jellyfin.html:
```diff
@@ -0,0 +1,49 @@
+            <div class="padded-left padded-right padded-bottom-page">
+                <form class="manualLoginForm" style="margin: 0 auto;">
+
+                    <h1 style="margin-top:1em;text-align: left;">Please enter your Jellyfin server address</h1>
```
All in-line styling should be moved to firstrun/style.css

cvium commented:
You should also remove database.txt

almet commented:
Awesome! I've got exactly the same need :-)

almet commented on .gitignore:
```diff
@@ -5,6 +5,7 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+.DS_Store
```
I think this should go in your global gitignore rather than in the project-specific gitignore (wich is supposed to stay OS-agnostic), no?

cvium replied:
This is much easier for contributors though

almet replied:
It's your call, really. If you see this as a valuable addition, why not :-)

skaro13 replied:
On second thought keeping it OS agnostic is good. I guess you're right this should go in global.

skaro13 commented:
I fixed everything suggested and fixed a few more things too: 
- Updated Licence value in package.json
- Removed duplicate dependencies between dev/prod
- Fixed a but where form would not submit when pressing enter instead of clicking save

skaro13 commented:
I also removed splash.html which is no longer required. BTW since this is a cross-platform app should we remove the windows_quick_install.bat file? It's basically just a clone and npm install which is already documented in the README.

skaro13 commented:
<img width="1392" alt="Capture d’écran, le 2019-04-28 à 23 57 00" src="https://user-images.githubusercontent.com/8635747/56875882-82071000-6a11-11e9-8e2f-a90e523b3bad.png">
I updated the login UI so it's the same as the Android apps.

dkanada commented on firstrun/style.css:
```diff
@@ -0,0 +1 @@
+body,html{margin:0;padding:0;height:100%}.backgroundContainer{position:fixed;top:0;left:0;right:0;bottom:0;contain:strict}html{line-height:1.35}.layout-mobile,.layout-tv{-webkit-touch-callout:none;-webkit-user-select:none;-khtml-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none}body{overflow-y:scroll!important;overflow-x:hidden;background-color:transparent!important;-we ...
```
not sure this should be minimized

dkanada replied:
@skaro13 thoughts?

skaro13 replied:
I agree since any other css for this project seems to be unminified.
